portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,6 +39,17 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Create the query and prepared query to load comment entities from database
+    Query query = new Query("Comment");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+    
+    // Add all comment contents to the msgList  
+    ArrayList<String> msglist = new ArrayList<String>();
+    for (Entity comment:results.asIterable()) {
+        String commentMsg = (String)comment.getProperty("content");
```

Comment:
Spacing: https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

Reply:
Just to double check: should I add the declaration of ArrayList<String> on line 46 instead of putting a new empty line in between. I just thought from line 47 to line 52 are doing one blocked feature